### PR TITLE
Fix for unavailable nvme-cli package on 6.x

### PR DESCRIPTION
**Pull request by sireeshapotnuru**
on RHEL/Centos/Oracle 6.x series "yum install nvme-cli"  is failed, due to  no nvme-cli package is available. So, Handled a check in the code to download and install the latest nvme package

### CODE REVIEW

**Comment by sireeshapotnuru** [comment]
[LISAv2 Test Results Summary]
Test Run On           : 05/14/2019 09:03:02
ARM Image Under Test  : RedHat : RHEL : 6.10 : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:38

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NVME                 PERF-NVME-4K-IO                                                                   PASS                27.65 
	Mode=randread : block_size=4K q_depth=8 iops=102849 
	Mode=randread : block_size=4K q_depth=16 iops=205000 
	Mode=randread : block_size=4K q_depth=32 iops=305681 
	Mode=randread : block_size=4K q_depth=64 iops=360658 
	Mode=randread : block_size=4K q_depth=128 iops=374366 
	Mode=randread : block_size=4K q_depth=256 iops=376037 
	Mode=randread : block_size=4K q_depth=512 iops=373584 
	Mode=randread : block_size=4K q_depth=1024 iops=373907 
	Mode=randread : block_size=4K q_depth=2048 iops=378397 
	Mode=randwrite : block_size=4K q_depth=8 iops=119052 
	Mode=randwrite : block_size=4K q_depth=16 iops=205671 
	Mode=randwrite : block_size=4K q_depth=32 iops=274901 
	Mode=randwrite : block_size=4K q_depth=64 iops=281385 
	Mode=randwrite : block_size=4K q_depth=128 iops=283808 
	Mode=randwrite : block_size=4K q_depth=256 iops=282741 
	Mode=randwrite : block_size=4K q_depth=512 iops=283474 
	Mode=randwrite : block_size=4K q_depth=1024 iops=282253 
	Mode=randwrite : block_size=4K q_depth=2048 iops=281889 
	Mode=read : block_size=4K q_depth=8 iops=109798 
	Mode=read : block_size=4K q_depth=16 iops=223703 
	Mode=read : block_size=4K q_depth=32 iops=332213 
	Mode=read : block_size=4K q_depth=64 iops=364426 
	Mode=read : block_size=4K q_depth=128 iops=369381 
	Mode=read : block_size=4K q_depth=256 iops=377106 
	Mode=read : block_size=4K q_depth=512 iops=377160 
	Mode=read : block_size=4K q_depth=1024 iops=377170 
	Mode=read : block_size=4K q_depth=2048 iops=377156 
	Mode=write : block_size=4K q_depth=8 iops=117301 
	Mode=write : block_size=4K q_depth=16 iops=208543 
	Mode=write : block_size=4K q_depth=32 iops=285979 
	Mode=write : block_size=4K q_depth=64 iops=239516 
	Mode=write : block_size=4K q_depth=128 iops=251863 
	Mode=write : block_size=4K q_depth=256 iops=257651 
	Mode=write : block_size=4K q_depth=512 iops=251497 
	Mode=write : block_size=4K q_depth=1024 iops=256379 
	Mode=write : block_size=4K q_depth=2048 iops=260124

**Review by lizzha — APPROVED** [review]

**Review by iamshital — APPROVED** [review]

File: Testscripts/Linux/perf_fio.sh
Comment (chvalean) [inline Testscripts/Linux/perf_fio.sh]: what this all new code is missing is a basic check if nvme-cli binary already exist on the system.
install_package is fine, at best it will update the package to latest version, but you're doing here a lot of work - deps, compile without a check if nvme-cli might already be on the system.
Please consider adding that, maybe another check in the if loop on line 180 along with the distro version being 6.x

Comment (sireeshapotnuru) [reply Testscripts/Linux/perf_fio.sh]: Added a check if nvme is already installed and also added a new function check_package which checks availability of package
Moved this block to a function in utils since it is common for all the nvme tests

**Comment by sireeshapotnuru** [comment]
@chvalean please approve this PR

File: Testscripts/Linux/utils.sh
Comment (lizzha) [inline Testscripts/Linux/utils.sh]: should be install_nvme_cli?

**Comment by iamshital** [comment]
@sireeshapotnuru Please post the results of all the affected tests.

**Comment by chvalean** [comment]
Please wait for @adriansuhov sign-off on this code

File: Testscripts/Linux/utils.sh
Comment (sireeshapotnuru) [reply Testscripts/Linux/utils.sh]: Thats Correct .. Changed function name to install_nvme_cli

**Comment by iamshital** [comment]
@sireeshapotnuru Resolve the confilcts @adriansuhov Please have it reviewed as Chris asked.

**Comment by sireeshapotnuru** [comment]
> @sireeshapotnuru Resolve the confilcts @adriansuhov Please have it reviewed as Chris asked.

Resolved the merge confilcts

**Pull request merged**
